src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -1318,7 +1317,7 @@ export default class Blockchain extends Emittery.Typed<
     // get storage data given a set of keys
     const options = {
       disableMemory: true,
-      disableStack: false,
+      disableStack: true,
```

Comment:
Didn't seem like we needed to trace the stack, and the tests still passed. I didn't look into this any further. :sweat_smile: